cli/heartbeat.js

```diff
@@ -52,6 +53,12 @@ const launchTime = Math.floor(new Date() / 1000);
 
 let uid = null;
 
+let sysStateCount = { "normal": 0, "overheated": 0 };
+let overheatedThresholds = null;
+(async function() {
```

Comment:
minor: run setInterval at the end of this function instead of main-loop

Reply:
Fixed as suggested